test_pokemon.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import numpy as np
+
+#from pkmn import sigmoid
+#from pkmn import relu_hidden_layer
+#from pkmn import policy_forward
+import pkmn
+import pickle
+
+class TestPong(unittest.TestCase):
```

Comment:
Why is this called TestPong?

Reply:
The original file was for testing Pong, and I hadn't bothered to rename it yet. (I have now bothered).